src/install-devtools.js

```diff
@@ -1,6 +1,8 @@
+const key = '__MICROCOSM_DEVTOOLS_GLOBAL_HOOK__'
```

Comment:
Bug caught by running a prod build! the `__` here was picking up our regex, so dev mode was broken in the minified build.